packages/tags/README.md

```diff
@@ -1,34 +1,40 @@
-## Description
+## Overview
 
 `<sp-tags>` elements contain a collection of `<sp-tag>` elements and allow users to categorize content. They can represent keywords or people, and are grouped to describe an item or a search request.
 
+[View the design documentation for this component.](https://spectrum.adobe.com/page/tag/)
+
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/tags?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/tags)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/tags?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/tags)
 [![Try it on Stackblitz](https://img.shields.io/badge/Try%20it%20on-Stackblitz-blue?style=for-the-badge)](https://stackblitz.com/edit/vitejs-vite-pbqsxwt5)
 
-```
+```zsh
 yarn add @spectrum-web-components/tags
 ```
 
 Import the side effectful registration of `<sp-tags>` or `<sp-tag>` via:
 
-```
+```js
 import '@spectrum-web-components/tags/sp-tags.js';
 import '@spectrum-web-components/tags/sp-tag.js';
 ```
 
 When looking to leverage the `Tags` or `Tag` base classes as a type and/or for extension purposes, do so via:
 
+```js
+import { Tags, Tag } from '@spectrum-web-components/tags';
 ```
-import {
-    Tags,
-    Tag
-} from '@spectrum-web-components/tags';
-```
 
-## Example
+### Anatomy
+
+Tags are created from the following parts:
+
+- **Tags**: The container component (`<sp-tags>`) that manages a collection of `Tag` elements.
+- **Tag**: The individual tag element (`<sp-tag>`) that represents a single tag. [Read more about the `Tag` component.](/components/tag/)
+
+### Examples
 
 ```html-live
```

Comment:
Is there a preference between `html` and `html-live`? I see more `html` in the components we've already done but am not sure if there's a specific preference.